docs/data/material/customization/dark-mode/dark-mode.md

```diff
@@ -251,7 +265,50 @@ Apply styles for a specific mode.
 #### Arguments
 
 - `mode` (`'light' | 'dark'`) - The mode for which the styles should be applied.
-- `styles` (`CSSObject`) - An object which contains the styles to be applied for the specified mode.
+- `styles` (`CSSObject`) - An object that contains the styles to be applied for the specified mode.
+
+#### Overriding applyStyles
+
+You can override `theme.applyStyles()` with a custom function to have complete control over it.
```

Comment:
Nitpick: to be more specific about what's meant by "have complete control over it."

```suggestion
You can override `theme.applyStyles()` with a custom function to gain complete control over the values it returns.
```